packages/content-management/table_list_view_table/src/components/tag_filter_panel.tsx

```diff
@@ -43,7 +43,8 @@ const saveBtnWrapperCSS = css`
   width: 100%;
 `;
 
-interface Props {
+interface Context {
+  enabled: boolean;
```

Comment:
I don't think we need to pass the `enabled` and handle it on L61 as the consumer (the panel) does not render if tagging is not enabled.

Reply:
Good call - commit [ed518a2](https://github.com/elastic/kibana/pull/188774/commits/ed518a29e4e89a79da4e0c8d6e15dd80a6d4e9f1) for changes